node/src/main/scala/coop/rchain/node/api/ReplGrpcService.scala

```diff
@@ -40,6 +42,16 @@ import storage.StoragePrinter
 private[api] class ReplGrpcService(runtime: Runtime[Task], worker: Scheduler)
     extends ReplGrpcMonix.Repl {
 
+  private[this] def interpreter() =
+    for {
+      costAlg <- CostAccounting.of[Task](Cost.Max)
+      cost    = loggingCost[Task](costAlg, noOpCostLog)
+      result = {
+        implicit val c = cost
+        Interpreter[Task]
+      }
+    } yield result
```

Comment:
I think Joe's comment applies here as well, and in some other places too. Basically, we now need `runtime.cost` to instantiate interpreter and should only create a standalone `cost` for the interpreter in test utils and similar places.